src/elements/content-sidebar/SidebarPanels.js

```diff
@@ -41,10 +41,13 @@ type Props = {
     metadataSidebarProps: MetadataSidebarProps,
     onVersionChange?: Function,
     onVersionHistoryClick?: Function,
-    refreshIdentity?: boolean,
     versionsSidebarProps: VersionsSidebarProps,
 };
 
+type ElementRefType = {
+    current: null | Object,
```

Comment:
Can we specify `{ refresh: () => {} }` here, as we did in ContentPreview?

Reply:
Unfortunately flow throws an error with that structure, thats why I just defined a type here instead of in the elements flowType file, since the ContentPreview example doesn't share the same type shape. Its likely due to the async loading of these sidebar components: https://github.com/box/box-ui-elements/pull/1561/files#diff-ea8b257187d0274f39a99eef07fc779bR59